README.md

```diff
@@ -312,10 +317,42 @@ The constructor of `MainWindow` class can be modified further to utilize WAM for
         
         //this is the currently recommended way to log MSAL message. For more info refer to https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/wiki/logging
         .WithLogging(new IdentityLogger(EventLogLevel.Warning), enablePiiLogging: false) //set Identity Logging level to Warning which is a middle ground
+        .WithClientCapabilities(new string[] { "cp1" }) //client capabilities for CAE - https://learn.microsoft.com/azure/active-directory/develop/app-resilience-continuous-access-evaluation?tabs=dotnet
         .Build();
 ```
 
 Refer to [MSAL WAM](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/wiki/wam#to-enable-wam-preview) for more details on how to write code for this.
+
+### Process the CAE challenge from Microsoft Graph
+
+To process the CAE challenge from Microsoft Graph, the controller actions need to extract it from the `wwwAuthenticate` header. It is returned when MS Graph rejects a seemingly valid Access tokens for MS Graph. For this you need to:
+
+1. To use `.WithClientCapabilities` method when creating Public Client app
+
+   ```csharp
+    .WithClientCapabilities(new string[] { "cp1" }) //client capabilities for CAE - https://learn.microsoft.com/azure/active-directory/develop/app-resilience-continuous-access-evaluation?tabs=dotnet
+   ```
+
+2. Catch `ServiceException` thrown by Graph API, try to obtain new token interactively, create Graph API client with the new token:
+
+   ```csharp
+     catch (ServiceException ex) when (ex.Message.Contains("Continuous access evaluation resulted in claims challenge"))
```

Comment:
We should not take a dependency on an exception message, because those can change without warning. Is there nothing else in the service exception that tells us it is for CAE? Like some sort of status code? If there isn't just rely on the header parsing.

Reply:
I'd protested about this model with Graph sDK team, and I'd have preferred the raw exception bubbled as-is to us, but that dint work out. ServiceException is super generic and no useful inner-exception to help with matters either. 

@aremo-ms , could you take a full dump of this exception for us please, like all properties , including any inner exceptions